rest/config.go

```diff
@@ -970,6 +970,10 @@ func (dbConfig *DbConfig) validateVersion(ctx context.Context, isEnterpriseEditi
 		}
 	}
 
+	if dbConfig.CORS != nil {
+		_, err := hostOnlyCORS(dbConfig.CORS.Origin)
+		base.WarnfCtx(ctx, "The cors.origin contains values that will be ignored: %s", err)
```

Comment:
Technically we're still going to try to string match against these when constructing Access-Control-Allow-Origin for non-ws requests, so the error is a bit misleading.  We could indicate what we're expecting (origin is not a well-formed URL)", and maybe change the 'will' to 'may' (to indicate that we don't know for sure that clients aren't going to send a malformed 'Origin' header for non-ws requests).